packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -9,15 +9,20 @@ import a11yPlugin from 'colord/plugins/a11y';
  * WordPress dependencies
  */
 import { SVG } from '@wordpress/components';
-import { useCallback, useMemo } from '@wordpress/element';
+import { useCallback, useMemo, createContext } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import transformStyles from '../../utils/transform-styles';
+import { store as blockEditorStore } from '../../store';
+import { unlock } from '../../lock-unlock';
 
 extend( [ namesPlugin, a11yPlugin ] );
 
+export const updateStyleContext = createContext( () => {} );
```

Comment:
I missed this one. I think this context is no longer needed.

Reply:
Sorry, badly rebased.